test/em_real/wind-turbine-1.tbl

```diff
@@ -0,0 +1 @@
+../../run/wind-turbine-1.tbl
```

Comment:
This file should not be part of the commit

Reply:
OK. Thanks.